pkgs/tools/security/truecrack/default.nix

```diff
@@ -33,6 +33,7 @@ gccStdenv.mkDerivation rec {
   meta = with lib; {
     description = "TrueCrack is a brute-force password cracker for TrueCrypt volumes. It works on Linux and it is optimized for Nvidia Cuda technology.";
     homepage = "https://gitlab.com/kalilinux/packages/truecrack";
+    broken = cudaSupport;
```

Comment:
CC @ethancedwards8 I couldn't build `truecrack-cuda` neither in `nixpkgs-unstable` nor in `master`, neither with cuda10 or cuda11

Reply:
Just saw this. Thanks for the information. Sorry for not responding earlier. Currently, I'm no longer using NixOS full time, nor am I using Truecrack, so I'm afraid I'm not willing to fix it. I believe building without cuda support works, right? If so, I don't see much of a reason to worry. If someone was using cuda support, I imagine someone would have created an issue.

Reply:
👍🏻 @ethancedwards8  No problem, I'm only trying to keep track of what works and what doesn't in nixpkgs when it comes to acceleation.

You also can remove yourself from the maintainer list for the package to stop receiving notifications about it